revpi-hat-eep/src/bin/revpi-eep.rs

```diff
@@ -209,12 +207,66 @@ fn main() {
         }
     };
 
-    let edate = match cli.edate {
-        Some(edate) => edate,
-        None => chrono::Local::today().naive_local(),
+    let serial = if let Some(serial_cli) = cli.serial {
+        if let Some(serial_config) = config.serial {
```

Comment:
Looks a little bit too complex, but haven't a better proposal either.

Reply:
It could be simpler, but then the error message wouldn't be this expressive.